experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -330,6 +330,9 @@ export class HttpInstrumentation extends InstrumentationBase<Http> {
       'response',
       (response: http.IncomingMessage & { aborted?: boolean }) => {
         this._diag.debug('outgoingRequest on response()');
+        if (request.listenerCount('request') <= 0) {
+          response.resume();
+        }
```

Comment:
If we are the only listener, we have to make sure to read the response data. Otherwise the connection is never closed.